code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -214,45 +220,46 @@
 
 	var/old_heat_capacity = air.heat_capacity()
 	var/reaction_energy
-	var/mediation = 100*(air.heat_capacity()-(cached_gases[/datum/gas/plasma][MOLES]*cached_gases[/datum/gas/plasma][GAS_META][META_GAS_SPECIFIC_HEAT]))/(air.total_moles()-cached_gases[/datum/gas/plasma][MOLES]) //This is the average heat capacity of the mixture,not including plasma.
+
+	var/mediation = FUSION_MEDIATION_FACTOR*(air.heat_capacity()-(cached_gases[/datum/gas/plasma][MOLES]*cached_gases[/datum/gas/plasma][GAS_META][META_GAS_SPECIFIC_HEAT]))/(air.total_moles()-cached_gases[/datum/gas/plasma][MOLES]) //This is the average heat capacity of the mixture,not including plasma.
 	var/gas_power = 0
 	for (var/id in cached_gases)
 		gas_power += cached_gases[id][GAS_META][META_GAS_FUSION_POWER]*cached_gases[id][MOLES]
```

Comment:
yeah thats a problem I noticed too, but I'm not sure how to change it

Reply:
I would suggest an efficiency factor determined by a function based on the ratio of plasma : other gases, being this the greater the closest to the 1:1 ratio. But that's because I like functions, I don't know how them work in this code. The function would be: y=60^[-(x^2/0.6)] where x is: (moles of plasma - moles of other gases non plasma)/total moles. It's a pretty graph. 

Or you know, you could do the "(moles of plasma - moles of other gases non plasma)/total moles", make that positive and make it the efficiency factor, but thats no fun because no graphs involved.

Gas power can be as it its now for less headaches, but being multiplied in all its instances by this efficiency factor.

Reply:
the idea of the reaction efficiency and the graph are pretty cool, i've implemented those and i'm gonna test them in a bit before i commit

Reply:
Ok, but consider changing the numbers in the mediation var as well so it's less overpowered, since this is actually a nerf

Reply:
yeah i'm testing different numbers for it right now